NetKAN/NuclearPropulsionDrive.netkan

```diff
@@ -0,0 +1,13 @@
+{
+    "$kref": "#/ckan/spacedock/470",
+    "identifier": "NuclearPropulsionDrive",
+    "license": "MIT",
+    "spec_version": "v1.4",
+    "install": [
+        {
+            "find": "parts",
+            "install_to": "GameData/NuclearPropulsionDrive",
+            "comment": "Zip has no modname folder"
```

Comment:
Yup. If we're lucky, the author will never push another release. Did you see what fuel the drive uses?